styles/dictionaries/wasmCloud-dictionary.dic

```diff
@@ -1,7 +1,10 @@
 
 Akka
-GitHub/M
 coroutine/S
+GitHub/M
+JetStream/M
+Kubernetes/M
 stateful
-wasmCloud/M
 WASI
+wadm
+wasmCloud/M
```

Comment:
Can't remember but were we not going to "wasmcloud" everywhere? is that worth including here?

Reply:
wasmCloud everywhere!